kaal-domain/build.gradle.kts

```diff
@@ -17,12 +17,6 @@ dependencies {
     testImplementation(Dependencies.Test.kotlinTest)
 }
 
-tasks.withType<KotlinCompile> {
-    kotlinOptions {
-        jvmTarget = "1.8"
-    }
-}
-
```

Comment:
Why did you remove this?

Reply:
Because it is not possible to compile it when we set jvmTarget to 1.8 here. I was following the same as it was done in maps

Reply:
Apply changes from f8d4fda1c1f0ab9c8eb3f65e0c035782472765c5

Reply:
Ok, done